valhalla/baldr/curl_tilegetter.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <valhalla/baldr/curler.h>
+#include <valhalla/baldr/tilegetter.h>
+
+#include <string>
+#include <utility>
+
+namespace valhalla {
+namespace baldr {
+
+/**
+ * Default implementation which uses libcurl and curler_pool_t.
+ */
+class curl_tile_getter_t : public tile_getter_t {
```

Comment:
The convention for class names in valhalla seem to be to use camel-case. Any particular reason this (and the class it derives from) uses snake-case instead? It would be good to standardize around a single convention for better readability.

Reply:
@mandeepsandhu thats not really true. its true in some namespaces (especially Odin) but less true in other ones.

Reply:
Frankly by the moment I wrote the base and derived I'd already seen many classes named in snake case here. Thanks for noticing, going to rewrite everything in camel case.

Reply:
Thats unfortunate. Yeah, I just spot checked only a few modules in `baldr` & `odin` to see if all classes were camel-cased.

So is there a preferred convention? (it seems to be to follow whatever existing convention is used in a file?)